src/stories/elements/MetadataKeyValueList/MeatadataKeyValueList.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import {
+  MetadataKeyValueList,
+  MetadataKeyValueListProps
+} from '../../../components/elements/metadataKeyValue/MetadataKeyValueList';
+
+const meta: Meta<MetadataKeyValueListProps> = {
+  title: 'Elements/MetadataKeyValueList',
+  component: MetadataKeyValueList
+};
+
+export default meta;
+
+const Template: StoryFn<typeof MetadataKeyValueList> = (args) => <MetadataKeyValueList {...args} />;
+
+export const ControlledDisplayFilterBasic: StoryFn<typeof MetadataKeyValueList> = Template.bind({});
+ControlledDisplayFilterBasic.args = {
+  metadataList: [
+    { key: 'key1', value: 'mockValue1', valueType: 'STRING', multiple: false },
+    { key: 'key2', value: '42', valueType: 'INTEGER', isAutoGenerated: true, multiple: false },
+    { key: 'key3', value: 'true', valueType: 'BOOLEAN', multiple: false }
+  ],
+  onDeleteHandler: (keyName:string)=>{console.log(`Metadata field with key name ${keyName} was deleted`)}
+};
```

Comment:
Same comment as in the metadata tags: I think this belongs inside the data-engine directory, WDYT?